recipe_scrapers/rezeptwelt.py

```diff
@@ -36,3 +102,13 @@ def description(self):
 
     def language(self):
         return self.soup.find("meta", {"property": "og:locale"})["content"]
+
+    def prep_time(self):
+        tag = self.soup.find(itemprop="performTime", content=nonempty)
+        return get_minutes(tag['content']) if tag else None
+
+    def equipment(self):
+        return [tag['content'] for tag in self.soup.find_all("meta", itemprop="tool", content=nonempty)]
```

Comment:
BeautifulSoup (`bs4` / `self.soup`) allows non-empty content filtering by passing a boolean `True` value, so I think we can simplify these methods slightly:

```suggestion
    def prep_time(self):
        tag = self.soup.find(itemprop="performTime", content=True)
        return get_minutes(tag['content']) if tag else None

    def equipment(self):
        return [tag['content'] for tag in self.soup.find_all("meta", itemprop="tool", content=True)]
```